newIDE/app/src/Utils/SerializableObjectCancelableEditor.js

```diff
@@ -64,13 +67,19 @@ export const useSerializableObjectCancelableEditor = ({
       if (!serializedElement) return;
 
       let continueCanceling = false;
+      const hasCancelBackdropPreference = backdropClickBehavior === 'cancel';
```

Comment:
should this `'cancel'` be stored in a variable in the preferences? To avoid regressions if one day we decided to change it?

Reply:
hmm, the variable is typed properly, so I assume we're fine in terms of catching a regression.
I also think it's clearer to use a string like this, rather than importing the constants everywhere we need them

Reply:
Oh yeah, I didn't check for types